index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap" rel="stylesheet">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+        <a class="logo">Saba Ahmad</a>
+          <nav id="items">
+        <i class="fa fa-bars"></i>
+     </nav>
+     </div>
+</header>
+<section>
+<div class="content">
+    <h1>I'm Saba <br> Glad to see <br> you!</h1>
```

Comment:
kindly note that for the text Glad to see and "you"  should use a header tag that is hx ( <h1>, <h2>, <h3>).This is the semantic correct way Thank you

Reply:
- Good job with your review 👏 . Please I'd suggest you should try to be clear nd concise when suggesting changes, this will help the author of the project understand what you mean better 😄 😄 
- I'd also recommend that you make use of emojis when suggesting changes, this will help you sound more friendly  👍 👍